forte/data/ontology/top.py

```diff
@@ -1101,6 +1065,15 @@ def __getstate__(self):
         if isinstance(state["_embedding"], np.ndarray):
             state["_embedding"] = list(self._embedding.tolist())
 
+        state["replace_back_operations"] = [
+            ((span.begin, span.end), text)
+            for span, text in self.replace_back_operations
+        ]
+        state["processed_original_spans"] = [
+            ((span1.begin, span1.end), (span2.begin, span2.end))
+            for span1, span2 in self.processed_original_spans
```

Comment:
Would it be easier if we change the `__setstate__` `__getstate__` for the `Span` class?